### PR TITLE
Cf create app

### PR DESCRIPTION
**Pull request by NimRegev**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cf-create-app/docs/

File: _docs/deployment/create-application.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+toc: true
+---
+
+
+
+Create applications that are fully GitOps compliant in the Codefresh UI, from generating the application configuration manifest, committing it to Git, and then syncing and deploying to the cluster. 
+
+Creating an application in Codefresh includes application definitions and application configuration settings. The create application wizard guides you through the process. 
+
+For how-to information, see [How to: Create an application](#how-to-create-an-Argo-CD-application)
```
Comment (kostis-codefresh) [inline _docs/deployment/create-application.md:14]: This doesn't seem to link anywhere

File: _docs/deployment/create-application.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+description: ""
+group: deployment
+toc: true
+---
+
+
+
+Create applications that are fully GitOps compliant in the Codefresh UI, from generating the application configuration manifest, committing it to Git, and then syncing and deploying to the cluster. 
+
+Creating an application in Codefresh includes application definitions and application configuration settings. The create application wizard guides you through the process.
```
Comment (kostis-codefresh) [inline _docs/deployment/create-application.md:12]: Include a screenshot of a deployed application here.

File: _docs/deployment/create-application.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+#### Destination
+The cluster and namespace to which to deploy the application.  
+{::nomarkdown}<ul><li><b>Cluster</b>: The cluster to which to deploy the application, defined as a <b>URL</b> or as the user-defined display <b>NAME</b>.</li> <li><b>Namespace</b>: The namespace in the cluster to which to deploy the application.</li> </ul> {:/}
+ 
+#### Sync Settings
+{::nomarkdown}<b>Sync Policy</b>: The synchronization policy to apply when there are differences between the desired state in Git and the actual state in the cluster.</br><ul><li><b>Manual</b>: Manually sync the changes from the Argo CD UI. </li><li><b>Automatic</b>: Automatically sync changes, with the following options if selected:<ul><li><b>Prune resources</b>:When selected, removes legacy resources that do not exist currently in Git. </li><li><b>Self heal</b>: When selected, always enforces a sync to the desired state in Git, if and when there is a change to the actual state in the cluster. See <a href="https://argo-cd.readthedocs.io/en/stable/user-guide/auto_sync/#automatic-self-healing" target="_blank">Automatic self-healing</a>.</li></li></ul> {:/}
```
Comment (kostis-codefresh) [inline _docs/deployment/create-application.md:33]: Include a screenshot of where the sync settings are defined.

File: _docs/deployment/create-application.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+Create applications that are fully GitOps compliant in the Codefresh UI, from generating the application configuration manifest, committing it to Git, and then syncing and deploying to the cluster. 
+
+Creating an application in Codefresh includes application definitions and application configuration settings. The create application wizard guides you through the process. 
+
+For how-to information, see [How to: Create an application](#how-to-create-an-Argo-CD-application)
+
+
+
+### Application definitions
+Application definitions include the name, runtime, and the name of the YAML manifest. By default, the YAML manifest has the same name as that of the application.
```
[the remark this answers is not in the document]
Comment (NimRegev) [reply _docs/deployment/create-application.md:19]: yes, when you define the config and until the final commit

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cf-create-app/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cf-create-app/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cf-create-app/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cf-create-app/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/cf-create-app/docs/

**Pull request merged**